wcivf/apps/parishes/templates/parishes/includes/_card.html

```diff
@@ -0,0 +1,38 @@
+{% load humanize %}
+
+<div class="card" id="parishcouncil">
+
+<h1>
+    <span aria-hidden="true">🗳️</span>
+    {{ parish_council.council_name }}
+    <span aria-hidden="true">🗳️</span>
+</h1>
+
+<p>
+    {% if parish_council.is_contested %}
+    You will also be able to vote for {{ parish_council.ward_seats }} councillors to sit
+    {% else %}
+    You are also represented by {{ parish_council.ward_seats }} councillors
+    {% endif %}
+    on <a href="{{ parish_council.website }}">{{ parish_council.council_name }}</a>{% if parish_council.parish_ward_name %} {{ parish_council.parish_ward_name}}{% endif %}. 
+</p>
+
+<p>
+    {{ parish_council.council_name }} is a {{ parish_council.council_type }} Council.
```

Comment:
This line looks a bit odd in the screenshots, as the data contains the council type in the name.

I suggest we either drop this line, or change the data so that `council_name` is "Foo" and the `council_type` is "Town Council". Then we can change below to:

```
In 2020-2021, {{ parish_council.council_name }} {{ parish_council.council_type }} raised
```

etc

Reply:
What about Salisbury City council tho

Reply:
If it's tricky then I suggest we just drop the line. "Foo Town Council is a Town Council" just isn't useful :)

Reply:
If we drop the line, I could add the link you wanted to the first line of the next paragraph @pmk01? "Town/Parish Councils are the first tier of local government"

Reply:
Yes, that sounds good!